### PR TITLE
Grammarly

### PR DESCRIPTION
**Pull request by mpoulin**
Please have a look before I merge.

I make some grammar/punctuation fixes. Also, I deleted the first paragraph.  It didn't seem to fit the overall theme of this section.

### CODE REVIEW

File: docs/writing/reading.rst [hunk 1]
```diff
@@ -3,10 +3,6 @@ Reading Great Code
 
 .. image:: /_static/photos/34689452831_93d7fd0571_k_d.jpg
 
-One of the core tenets behind the design of Python is creating
-readable code. The motivation behind this design is simple: The number
-one thing that Python programmers do is read code.
-
```
Comment (dbader) [inline docs/writing/reading.rst]: I'm thinking this paragraph works quite well—did you want to remove it entirely?

File: docs/writing/reading.rst [hunk 2]
```diff
@@ -28,12 +24,12 @@ reading. Each one of these projects is a paragon of Python coding.
 - `Diamond <https://github.com/python-diamond/Diamond>`_
   Diamond is a python daemon that collects metrics
   and publishes them to Graphite or other backends.
-  It is capable of collecting cpu, memory, network, i/o, load and disk metrics.
+  It is capable of collecting cpu, memory, network, i/o, load, and disk metrics.
```
Comment (dbader) [inline docs/writing/reading.rst]: Should we use capitals for "CPU" and "I/O"?

**Comment by dbader** [comment]
@mpoulin Added some comments—merge at will Marc 😃

File: docs/writing/reading.rst (hunk 1 above)
Comment (mpoulin) [reply docs/writing/reading.rst]: I think this paragraph is a good intro to issues of style, formatting, and PEP-8, but I think this section is about a different topic.

File: docs/writing/reading.rst (hunk 2 above)
Comment (mpoulin) [reply docs/writing/reading.rst]: Agreed, CPU and I/O should be upper case.

File: docs/writing/reading.rst (hunk 1 above)
Comment (mpoulin) [reply docs/writing/reading.rst]: So yes, I would remove it.

Comment (dbader) [reply docs/writing/reading.rst]: 👍 Makes sense!

**Pull request merged**